documentation/api/query/v4/pql.markdown

```diff
@@ -387,6 +387,10 @@ starts with `web`:
 
     facts[name, count(value)] { certname ~ "^web.*" group by name }
 
+Grouping on function results is also supported:
+
+    reports[count(), to_string(receive_time, DAY)]{group by to_string(receive_time, DAY)}
```

Comment:
ooh, bareword support for text args ... not sure I'm a fan ... right now unquoted-barewords in our language only represent functions, entities, fields ... this is none of those, it kind of changes the semantics a little. Up until now though our function args are pretty much only those things, so I get why this is easier but ... there are cases where users are going to have to provide spaces in their formatting ... so we'll need quotes anyway. Bare-word arbitrary string support in this place and not others is just going to be confusing also. I think this is a string argument, and should always be quoted basically.